inc/class-wpseo-replace-vars.php

```diff
@@ -1163,7 +1233,11 @@ private function is_not_prefixed( $replacement_variable ) {
 	 * @return array The formatted replacement variable.
 	 */
 	private function format_replacement_variable( $replacement_variable ) {
-		return array( 'name' => $replacement_variable, 'value' => '' );
+		$label = '';
+		if ( ! $this->is_not_prefixed( $replacement_variable ) ) {
```

Comment:
This line is pretty hard to read. It says: `if not is not prefixed`. Can you change this somehow.